pyface/data_view/tests/test_abstract_value_type.py

```diff
@@ -85,6 +85,16 @@ def test_get_image(self):
         result = value_type.get_image(self.model, [0], [0])
         self.assertEqual(result.name, "image_not_found")
 
+    def test_has_check_state(self):
+        value_type = ValueType()
+        result = value_type.has_check_state(self.model, [0], [0])
+        self.assertFalse(result)
+
+    def test_get_check_state(self):
+        value_type = ValueType()
+        result = value_type.get_check_state(self.model, [0], [0])
+        self.assertEqual(result, CheckState.CHECKED)
```

Comment:
can we also add a new trivial test here that test that `set_check_state` raises the expected error?